packages/cardpay-sdk/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Hassan Abdel-Rahman @habdelra",
   "main": "./dist/esm/index.js",
   "types": "./dist/index.d.ts",
+  "browser": "browser.js",
```

Comment:
im tempted to remove this. Because consumers of the sdk may look for different keys depending on their target. I think webpack default behaviour may be to prefer "browser" over "main", but i think there are complex set of rules that decide this. 

One solution is to use the "resolve.mainFields" which specify the priority. 

For my purposes, since I can point to the correct file (even using unpkg), there is no need to have this key.

Reply:
i think that the mobile app was relying on this. you might wanna make sure that you can still run the mobile app without this first.